operator/pkg/model/translation/envoy_listener_test.go

```diff
@@ -48,6 +48,19 @@ func TestNewHTTPListener(t *testing.T) {
 
 	})
 
+	t.Run("without TLS with Proxy Protocol", func(t *testing.T) {
+		res, err := NewHTTPListener("dummy-name", "dummy-secret-namespace", nil, WithProxyProtocol())
+		require.Nil(t, err)
+
+		listener := &envoy_config_listener.Listener{}
+		err = proto.Unmarshal(res.Value, listener)
+		require.Nil(t, err)
+
+		require.Equal(t, "dummy-name", listener.Name)
+		require.Len(t, listener.GetListenerFilters(), 2)
```

Comment:
Same comment as other places with this - could you please check that the second filter is the correct Proxy protocol filter?